tsconfig.base.json

```diff
@@ -17,6 +17,7 @@
     "strictPropertyInitialization": false,
     "paths": {
       "@near-wallet-selector/core": ["dist/packages/core"],
+      "@near-wallet-selector/modal-ui": ["dist/packages/modal-ui"],
```

Comment:
I think `core` can now point to `packages/core/src/index.ts` now. This is the improvement Rares talked about where we won't need to rebuild `core` every time we make changes to it - it'll just be when the modal is modified now